### DISCUSSION TITLE
How to get chain information by Chain ID

### DISCUSSION THREAD

**Discussion by liho00** [wevm/wagmi#2258] [Q&A]
I wanna set environment variable of the desired chain to be use in the entire app, 
I wish to have getChainByID kind of function to allow me get the chain detail that defined from the env vars
```
var mainnet = getChainByID(1) // mainnet
var arbitrumGoerli = getChainByID(421613) // arbitrumGoerli

console.log(arbitrumGoerli.name)
console.log(arbitrumGoerli.blockExplorers.url)

however currently only support this way...
import { mainnet, goerli, arbitrumGoerli } from 'wagmi/chains';

**Comment by Melancholism** [comment]
@liho00 you might find [this](https://github.com/wagmi-dev/viem/discussions/418#discussioncomment-5740428) helpful.
